pkg/util/nvidia/paths.go

```diff
@@ -82,14 +82,19 @@ func nvliblist(nvidiaDir string) ([]string, error) {
 // Paths returns list of nvidia libraries and binaries that should
 // be added to mounted into container if it needs NVIDIA GPUs.
 func Paths(nvidiaDir string, envPath string) ([]string, []string, error) {
+	ldConfig, err := exec.LookPath("ldconfig")
+	if err != nil {
```

Comment:
I think you should check for ErrNotFound here to be sure that `ldconfig` was not found indeed rather than any other error occurred. 
```
 ee, ok := err.(*exec.Error)
if ok && ee.Err == exec.ErrNotFound {}
```